src/PeerPoolMgr.h

```diff
@@ -32,6 +56,8 @@ class PeerPoolMgr: public AsyncJob
     explicit PeerPoolMgr(CachePeer *aPeer);
     ~PeerPoolMgr() override;
 
+    DetailedCodeContext::Pointer context; ///< the pool manager context
```

Comment:
Please rename -- there are too many contexts in this context :-). 

```suggestion
    DetailedCodeContext::Pointer codeContext;
```

I cannot think of any useful documentation for this self-documented declaration. The proposed description in current PR code reads like "foo is foo" to me. Existing official code has many self-documented codeContext data members, so I hope we do not have to describe this one either.

Reply:
Renamed.